public/app/features/alerting/unified/components/receivers/TemplatesTable.tsx

```diff
@@ -101,6 +101,16 @@ export const TemplatesTable: FC<Props> = ({ config, alertManagerName }) => {
                         icon="file-alt"
                       />
                     )}
+                    {contextSrv.hasPermission(permissions.create) && (
```

Comment:
I'd change the order of buttons to `Edit | Duplicate | Delete` so it is consistent with the order on the alert rules list page